src/Equinor.ProCoSys.IPO.Infrastructure/Repositories/ExportIPOs/ExportIpoRepository.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Transactions;
+using Dapper;
+using Equinor.ProCoSys.Common.Misc;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Microsoft.EntityFrameworkCore;
+
+
+namespace Equinor.ProCoSys.IPO.Infrastructure.Repositories.ExportIPOs;
+
+
+public class ExportIpoRepository : DapperRepositoryBase, IExportIpoRepository
+{
+    public ExportIpoRepository(IPOContext context) : base(context) { }
+
+    public async Task<List<Invitation>> GetInvitationsWithIncludesAsync(
+        List<int> invitationIds,
+        IPlantProvider plantProvider,
+        CancellationToken cancellationToken)
+    {
+        await using (var connection = Context.Database.GetDbConnection())
```

Comment:
for all using statements in this class, convert to using-declarations. As this:
![image](https://github.com/equinor/procosys-invitation-for-punch-out-api/assets/19545528/86de56b1-74ae-4229-ac92-9bfd89803acf)